java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java

```diff
@@ -134,8 +134,8 @@ public void put(String key, String value) {
                     if (this.config.getMessageCount().isPresent() &&
                         this.messagesSent >= this.config.getMessageCount().get()) {
                             // signal to main thread that all messages are sent, application can exit
```

Comment:
You hsould probably align this comment as well.

Reply:
Done